### PR TITLE
Improvements for conddb dump command

### PR DESCRIPTION
**Pull request by ggovi**
Goals of this PR:
-  Allow the usage of a library ( part of CMSSW release ) with built-in XML converters for specific payload classes. The payloads whose the XML dump is more frequent are belonging to this category. For the moment, only the RunInfo condFormat class has been added. Further addition will be addressed with specific PR by the owners of the concerned CondFormats. Only a one-line change PR is required for any addition. 
- Smoothen the XML conversion for payload classes NOT present in the built-in library.  A single plugin library is generated locally by the command,  per payload type. Once generated, the plugin library is kept for further usage, speeding up considerably the execution of multiple command requests.

### CODE REVIEW

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

File: CondCore/Utilities/interface/Payload2XMLModule.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#include <string>
```
Comment (davidlange6) [inline CondCore/Utilities/interface/Payload2XMLModule.h]: hi @ggovi - please add an include guard

**Comment by davidlange6** [comment]
-1
pending answers to small comments

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

File: CondCore/Utilities/interface/Payload2XMLModule.h (hunk 1 above)
Comment (ggovi) [reply CondCore/Utilities/interface/Payload2XMLModule.h]: true, just forgotten - done now.

**Comment by davidlange6** [comment]
+1

**Pull request merged**